src/plugins/creditor/credNodeStore.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+
+import {type CreditorNode, type NodeUuid, type NodeTagId} from "./creditorNode";
+import {JsonLog} from "../../util/jsonLog";
+
+export type NodeEntry = {|type: "NODE", entry: CreditorNode|};
+export type NodeDetails = {title: string, id: NodeUuid};
```

Comment:
```suggestion
export type NodeEntry = {|+type: "NODE", +entry: CreditorNode|};
export type NodeDetails = {|+title: string, +id: NodeUuid|};
```

Prefer making types exact and readonly unless we specifically want diff behavior. Wish this was the default tbh.